app/views/shipments/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h4>New Shipment</h4>
+
+<div class="well">
+  <%= form_for @shipment do |f| %>
+    <div class="form-group">
+      <%= f.label(:shipping_carrier) %>
+      <%= f.select(:shipping_carrier, Shipment::VALID_CARRIERS, { include_blank: "Please choose ..." }, { class: "form-control" }) %>
```

Comment:
The final parameter hash braces are unnecessary (and would actually fail in rubocop if it weren't erb).

You could instead do:
```erb
<%= f.select(:shipping_carrier, Shipment::VALID_CARRIERS, { include_blank: "Please choose ..." }, class: "form-control") %>
```

Though if you prefer it how it is, I think that's fine.